File: Problem191_Numberof1Bits.cpp
#include <iostream>
using namespace std;
class Solution {
public:
    int hammingWeight(int n) {
        int count = 0;
        while (n!=0) {
            if (n&1) {
                count++;
            } 
            n = n>>1;
        }
        return count;
    }
};
int main() {
    Solution solve;
    int n = 7;
    cout << solve.hammingWeight(n);
    return 0;
}

File: Problem191_Numberof1Bits.py
class Solution:
    def hammingWeight(self, n: int) -> int:
        count = 0
        while n!=0:
            if n&1:
                count+=1
            n=n>>1
        return count

if __name__ == "__main__":
    n = 1
    solve = Solution()
    print(solve.hammingWeight(n))